python/pickle/PyROOT_readtests.py

```diff
@@ -134,7 +134,10 @@ def test5ReadCustomTypes( self ):
 
    def test6ReadCustomTypes( self ):
       """[ROOT-10810] Test reading a RooDataSet with weights"""
-      ROOT.gEnv.SetValue("RooFit.Banner", 0)
+
+      if os.environ.get('ROOFIT') == 'False':
```

Comment:
What is being unpickled in this test are Roofit objects?

Reply:
Taking a look at the `writetests` file, it is because there was a Jira issue on pickling RooDataSets, which is why it's tested here to read them back in